pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/sink/PulsarSink.java

```diff
@@ -110,6 +108,22 @@ protected PulsarSinkProcessorBase(Schema schema) {
                 if (pulsarSinkConfig.getProducerSpec().getMaxPendingMessagesAcrossPartitions() != 0) {
                     builder.maxPendingMessagesAcrossPartitions(pulsarSinkConfig.getProducerSpec().getMaxPendingMessagesAcrossPartitions());
                 }
+                if (!pulsarSinkConfig.getProducerSpec().getDisableBatching()) {
+                    builder.enableBatching(true);
```

Comment:
The batching is enabled by default in the builder. So I don't think this change is able to disable batching.